src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,10 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+
+        int result;
+        result = ((result1 == bet1) && (result2 == bet2)) ? 2 :
+                ((((result1 <= result2) || (bet1 <= bet2)) && ((result1 >= result2) || (bet1 >= bet2)) && ((result1 != result2) || (bet1 != bet2))) ? 0 : 1);
```

Comment:
`mvn clean package` don't work, let's fix it )

Reply:
done